deploy/helm/templates/rbac/cluster_pod_required_role.yaml

```diff
@@ -14,123 +14,15 @@ aggregationRule:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: kubeblocks-lorry-pod-role
+  name: kubeblocks-kb-agent-pod-role
   labels:
     {{- include "kubeblocks.labels" . | nindent 4 }}
     app.kubernetes.io/required-by: pod
 rules:
+# this is needed to create role probe events
 - apiGroups:
   - ""
   resources:
   - events
   verbs:
   - create
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  verbs:
-  - create
-  - get
-  - list
-  - patch
-  - update
-  - delete
-- apiGroups:
-  - apps.kubeblocks.io
-  resources:
-  - clusters
-  verbs:
-  - get
-  - list
-- apiGroups:
-  - apps.kubeblocks.io
-  resources:
-  - clusters/status
-  verbs:
-  - get
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - get
-  - list
-- apiGroups:
-  - apps.kubeblocks.io
-  resources:
-  - opsrequests/status
-  verbs:
-  - get
-  - patch
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: kubeblocks-patroni-pod-role
-  labels:
-    {{- include "kubeblocks.labels" . | nindent 4 }}
-    app.kubernetes.io/required-by: pod
-rules:
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  verbs:
-  - create
-  - get
-  - list
-  - patch
-  - update
-  - watch
-  # delete is required only for 'patronictl remove'
-  - delete
-- apiGroups:
-  - ""
-  resources:
-  - endpoints
-  verbs:
-  - get
-  - patch
-  - update
-  - create
-  - list
-  - watch
-  # delete is required only for 'patronictl remove'
-  - delete
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: kubeblocks-backup-pod-role
```

Comment:
This role is also not needed since relavant code removed in https://github.com/apecloud/kubeblocks/pull/8203